Tests/AnimationViewTests.swift

```diff
@@ -61,4 +61,66 @@ final class AnimationViewTests: XCTestCase {
     wait(for: [expectation], timeout: 1.0)
   }
 
+  func testPlayFromFrameToFrame() {
```

Comment:
Pulled in this test case from https://github.com/airbnb/lottie-ios/pull/1892 -- thanks @nevil!